sebs/utils.py

```diff
@@ -200,3 +208,34 @@ def handler(x, y):
         sys.exit(signal.SIGINT)
 
     signal.signal(signal.SIGINT, handler)
+
+class ColoredPrinter:
+    SUCCESS = "\033[92m"
+    STATUS = "\033[94m"
+    WARNING = "\033[93m"
+    ERROR = "\033[91m"
+    BOLD = "\033[1m"
+    END = "\033[0m"
+
+    def __init__(self, logging_instance):
+        self.logging_instance = logging_instance
+
+    def debug(self, message):
+        self._print(message, ColoredPrinter.STATUS)
+    
+    def info(self, message):
+        self._print(message, ColoredPrinter.SUCCESS)
+
+    def warning(self, message):
+        self._print(message, ColoredPrinter.WARNING)
+
+    def error(self, message):
+        self._print(message, ColoredPrinter.ERROR)
+
+    def critical(self, message):
+        self._print(message, ColoredPrinter.ERROR)
+
+    def _print(self, message, color):
+        timestamp = datetime.datetime.now().strftime("%H:%M:%S.%f")
+        self.logging_instance.info(message)
```

Comment:
I think that is the last remaining issue - right now, the output is duplicated because we have defined `StreamHandler` above. I think your approach of printing to `logging_instance` is correct because we want to also write to file. But we should now disable `StreamHandler` and configure the `ColorPrinter` with `verbose` flag to decide if `debug` should be printed or not,

Reply:
I have amended `ColoredPrinter` to `ColoredWrapper`, which would only propagate logging instructions to `_logger` if `filename` in `LoggingHandler` is present. Moreover, `StreamHandler` is removed, and its functions are implemented in `ColoredWrapper` now.